drivers/input/ajoystick.c

```diff
@@ -380,14 +344,12 @@ static int ajoy_open(FAR struct file *filep)
 
   ajoy_enable(priv);
 
+  leave_critical_section(flags);
```

Comment:
Let's move this after `filep->f_priv = (FAR void *)opriv;`

Reply:
Done.